recipe/meta.yaml

```
@@ -11,25 +11,33 @@ source:
 
 build:
   number: 0
-  skip: true
+  skip: true # [py<37] 
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - python
+    - pip
+    - wheel
     - setuptools >=41
   run:
     - python
+    - importlib-metadata  # [py<38]
 
 test:
   imports:
     - paste
     - paste.deploy
```

There's no particularly hard and fast rules about what to `import` here.  My tooling leans into what you might see if you looked in the tests directory and did something like:

```
grep -rE '(from|import) +(pytest|unittest)' tests | awk '{print $2}' | sort -u
```

which would give you:

```suggestion
    - paste.deploy
    - paste.deploy.config
    - paste.deploy.converters
    - paste.deploy.util
```